sdcm/report_templates/results_sl_workloads.html

```diff
@@ -0,0 +1,48 @@
+{% extends 'results_base.html' %}
+{% block body %}
+    <h3>
+        <span>System under test </span>
+    </h3>
+    <div>
+        <ul>
+            <li>Scylla version: {{ scylla_version }}</li>
+            <li>Scylla ami id: {{ scylla_ami_id }}</li>
+            <li>Region: {{region}}</li>
```

Comment:
it could have been:
```suggestion
            <li>Scylla ami id: {{ scylla_ami_id }} ({{ region }})</li>
```

but LGTM